docs/doc-writer.php

```diff
@@ -71,10 +71,10 @@ function get_replace_position($lines) {
 }
 
 function build_documentation($docs) {
-    usort(
-        $docs,
-        function($a, $b) { return strcmp($a->methodName, $b->methodName); }
-    );
+//    usort(
```

Comment:
@asm89 Here's an example of what I'm talking about.  If you take a look at [do](
http://gemsbok.voryx.net/documentation/operators/do.html), the alphabetical ordering is `doOnCompleted doOnEach doOnError doOnNext`, which doesn't make sense, it should be `doOnEach doOnNext doOnError doOnCompleted`